ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -6,7 +6,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```

Comment:
The CI linter will report error for this change.

Reply:
Thanks for this catch! Reverted.